### PR TITLE
4 docker setup

### PR DESCRIPTION
**Pull request by AntonOstman**
Add docker with WASM. We need to check if the linting issue is an actual issue or false positive since adding file extension creates another linting problem

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -57,9 +57,11 @@
         ]
       }
     ],
-    "no-console": "off"
+    "no-console": "off",
+    "import/extensions": ["off"]
```
Comment (rajtarn) [inline .eslintrc.json:61]: @AntonOstman, I fixed eslint by disabling import/extensions check. I worked locally when the js file created by Emscripten was present, but not in GitHub when it was not. This check is not that important.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,27 @@
+FROM emscripten/emsdk:3.1.32
+WORKDIR /app
+
+COPY ./package.json .
+COPY ./vite.config.ts .
```
Comment (rajtarn) [inline Dockerfile:5]: @AntonOstman, are these copy instructions necessary if we copy everything att line 9?

**Review by rajtarn — APPROVED** [review]
Nice! 
@AntonOstman, @Dethcrvsh  one thing, we need to decide on the structure of the project. Where do we want to keep the cpp code? Now we have `src/cpp` and `libs/src/ `, and I think that @Dethcrvsh set up the linting and testing for `libs/src`?

File: .eslintrc.json (hunk 1 above)
Comment (AntonOstman) [reply .eslintrc.json:61]: nice!

**Review by Dethcrvsh — APPROVED** [review]
Looks good as far as I can see. I agree that the code might fit better in src/cpp and it wouldn't be too hard to change the setup for that.

**Pull request merged**